pkgs/applications/science/biology/spades/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, zlib, bzip2, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "SPAdes";
+  version = "3.13.1";
+
+  src = fetchurl {
+    url = "http://cab.spbu.ru/files/release${version}/${pname}-${version}.tar.gz";
+    sha256 = "0giayz197lmq2108filkn9izma3i803sb3iskv9hs5snzdr9p8ld";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ zlib bzip2 ];
+
+  doCheck = true;
+
+  postUnpack = ''
+    sourceRoot=''${sourceRoot}/src
+    echo Source root reset to ''${sourceRoot}
+  '';
```

Another way of doing this would be:
```suggestion
  sourceRoot = "${pname}-${version}/src";
```

---

fixed in 69e480b